### ISSUE TITLE
Applies `undefined` to calc

### ISSUE THREAD

**Issue by strarsis** [jhildenbiddle/css-vars-ponyfill#18]
The polyfill applies `undefined` in IE11:
```css
flex: calc(undefined / undefined);
````

Reproducible test case: https://codepen.io/strarsis/pen/JBLBax

```css
flex: calc(var(--img-width) / var(--img-height));
````
```html
style="--img-width:150;--img-height:150"
````

**Comment by jhildenbiddle** [comment]
Confirmed. Thanks for filing the issue!

**Comment by jhildenbiddle** [comment]
Fixed in 1.9.0, with automated tests added to the build.

Thanks again for filing the issue!
